auth0_flutter/lib/src/web/js_interop.dart

```diff
@@ -1,2 +1,79 @@
 // Here go the annotations –from the js package– that allow calling JS code
-// See https://pub.dev/packages/js#usage
\ No newline at end of file
+// See https://pub.dev/packages/js#usage
+
+// ignore_for_file: non_constant_identifier_names
+
+@JS('auth0')
+library auth0;
+
+import 'dart:js_util';
+
+import 'package:js/js.dart';
+
+@JS()
+@staticInterop
```

Comment:
Note that this is experimental, and we should use the current form of interop: `For now, static interop remains experimental and in development. We may make breaking changes in the future. We'll update this text when the new interop model matures and is considered stable.`. See https://pub.dev/packages/js#static-interop

Reply:
Thanks, I missed that. I'll swap it back

Reply:
Although we'll want to move back to it at some stage once it's stable.

Reply:
Fixed in https://github.com/auth0/auth0-flutter/pull/203/commits/1355315f9d09a217b5a6c4b93fa4a23e19ab8b08